### ISSUE TITLE
NODE USER FEEDBACK SESSION - 2019-05-16 - São Paulo - SP - Brazil - How does Node.js support your daily work?

### ISSUE THREAD

**Issue by ErickWendel** [nodejs/user-feedback#156]
**Event Name (pt-br)**: NodeBR #50 - Testes com Node.js
**Event Name (en-us)**: NodeBR #50 - Working with Tests with Node.js
**Link**: https://www.meetup.com/pt-BR/nodebr/events/260955295/

**Comment by khaosdoctor** [comment]
Live recorded feedback session at: https://youtu.be/xPdn0iTJ8MQ?t=3555

**Common points of discussion**

- Node-gyp often errors on version changes
  - Specially on node-sass and spdy packages
- Outdated or unmaintained packages often break when Node versions are updated
- Most people update node as soon as it is launched
- Most people are already using v12 or v10, no one is below those
- AWS makes updating past 10 impossible on lambda
- People tend to use latest LTS versions on staging environment for a few weeks before promoting it to production
- Tail call optimization was a problem raised to be improved

**Comment by brumone** [comment]
I've had node-gyp issues as well. I'm not sure what is the cause, but it always occurred when I ran a Vue.js project for the first time on a machine. The errors were always related to node-sass as well.

**Comment by IgorHalfeld** [comment]
Hmm.. this errors aren't related to Vue, it happen because `node-sass` package uses the lib-sass

**Comment by khaosdoctor** [comment]
@brumone Yeah, `node-sass` package is always erroring since v10 or earlier, `node-gyp` seems not to be able to build something on `lib-sass`. This is a clear representation of a package which needs to be maintained
